apps/api/src/common/hotcache.ts

```diff
@@ -0,0 +1,22 @@
+class CacheObject {
```

Comment:
I think, we could get away without this class, and do everything in the `HotCache class` instead.

Reply:
Javascript is not my strongest language, but I don't see an issue here.
I need structure that holds both the value and the update time.